### PR TITLE
UNET Model for high resolution CamVid images

### PR DESCRIPTION
**Pull request by ermanok**
commit includes:
- Camvid dataset modifications to get folded images
- Large UNet model for folded dataset
- Required training scripts for UNet model with CamVid dataset
- Modification in train.py to disable/limit accuracy calculation on the training set.

### CODE REVIEW

**Review by MaximGorkem — CHANGES_REQUESTED** [review]
"schedule-camvid.yaml" is missing. The training script requires it.

Other than that, it looks good.

**Review by vicloginov — CHANGES_REQUESTED** [review]
schedule-camvid.yaml file is missing. I used this for better accuracy:
**schedule-camvid.yaml:**
lr_schedulers:
  training_lr:
    class: MultiStepLR
    milestones: [50, 100, 140]
    gamma: 0.25

policies:
  - lr_scheduler:
      instance_name: training_lr
    starting_epoch: 0
    ending_epoch: 200
    frequency: 1

**qat_policy_camvid.yaml:**
start_epoch: 150
weight_bits: 8
